pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -177,6 +179,33 @@ func (pg *pgcontroller) getAnnotationsFromUpperRes(kind string, name string, nam
 	}
 }
 
+func (pg *pgcontroller) getMinMemberFromUpperRes(pod *v1.Pod) int32 {
+	minMember := int32(1)
+
+	for _, reference := range pod.OwnerReferences {
+		// Currently we assume the group-min-member annotation will be specified only once
+		if reference.Kind != "" && reference.Name != "" {
+			annotations := pg.getAnnotationsFromUpperRes(reference.Kind, reference.Name, pod.Namespace)
+			if minMemberAnno, ok := annotations[scheduling.VolcanoGroupMinMemberAnnotationKey]; ok {
+				minMemberFromAnno, err := strconv.Atoi(minMemberAnno)
+				if err != nil {
+					klog.Infof("Failed to convert minMemberAnnotation of Pod owners <%s/%s> into number, skipping",
```

Comment:
This klog info is not so clear, we should mention that remain minmember =1, or users will don't  know the value finally set.